src/main/java/org/terasology/buildings/BasicRasterTarget.java

```diff
@@ -41,7 +41,7 @@ public BasicRasterTarget(WorldProvider worldProvider,Rect2i area, BlockTheme blo
 
         Vector3i min = new Vector3i(affectedArea.minX(), -255, affectedArea.minY());
         Vector3i max = new Vector3i(affectedArea.maxX(), 255, affectedArea.maxY());
-        this.affectedRegion = Region3i.createFromMinMax(min, max);
+        this.affectedRegion = new BlockRegion(min, max);
```

Comment:
Since we create the vectors above from its components, we could just use those directly in the constructor